assignment.js

```diff
@@ -65,7 +72,11 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 2:')
 
 // Add your code below this line
+let numbers = [4,78,43,8, 123, 89, 675]
 
+let biggestNumber = Math.max(...numbers )
```

Comment:
@SOyeniyi801 nice use of the spread operator combined with `Math.max()`. I'm all for using these advanced approaches as long as you understand how it works.